janus_core/janus_types.py

```diff
@@ -0,0 +1,68 @@
+"""
+Module containing types used in Janus-Core
+"""
+
+from pathlib import Path, PurePath
+from typing import IO, List, Literal, Optional, Sequence, TypedDict, TypeVar, Union
+
+from ase import Atoms
+import numpy as np
+from numpy.typing import NDArray
+
+# General
+
+T = TypeVar("T")
+MaybeList = Union[T, List[T]]
```

Comment:
```suggestion
from typing import IO, Literal, Optional, Sequence, TypedDict, TypeVar, Union

from ase import Atoms
import numpy as np
from numpy.typing import NDArray

# General

T = TypeVar("T")
MaybeList = Union[T, list[T]]
```
I think `list` is preferable  `List` for [Python 3.9+](https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections)?

Reply:
if I understand correctly typing(which may not be the case, list is to be used when you do not care about what the list is of... while use list when you want to specify the type of elements. so I suspect shall be List[T]... but @ElliottKasoar and @oerc0122 I can be corrected.

Reply:
That is incorrect as of 3.9. `list` and most other standard Python types adopted the generic interfaces allowing you to specify sub-types without importing `typing`

Reply:
perfect so I will commit the changes.